pkg/operator/ceph/object/rgw.go

```diff
@@ -82,8 +82,12 @@ func (c *clusterConfig) createOrUpdateStore(realmName, zoneGroupName, zoneName s
 		return nil
 	}
 
-	if err = enableRGWDashboard(objContext); err != nil {
-		logger.Warningf("failed to enable dashboard for rgw. %v", err)
+	if c.clusterSpec.Dashboard.Enabled {
+		if !c.store.Spec.IsRGWDashboardEnabled() {
+			disableRGWDashboard(objContext)
```

Comment:
Do we need to reconcile if failed to enable or disable the dashboard? @travisn

Reply:
This PR will maintain the existing behavior to not fail/retry the reconcile if the dashboard config fails, so perhaps it's a question for a follow-up PR. The operator generally should fail the reconcile for most operations, but we also have a pattern of just logging a warning if it's considered a non-critical issue. I'd say this falls in that category of being non-critical, but we could certainly reconsider.